.github/workflows/test-all-custom-flags.yml

```diff
@@ -195,7 +195,7 @@ jobs:
 
       - name: Remove past BIT status comments on originating issue
         uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
-        if: (success() || failure()) && steps.set-send-pr-feedback.outputs.send_pr_feedback == 'true'
+        if: (success() || (failure() && steps.get-build-artifact.conclusion != 'failure')) && steps.set-send-pr-feedback.outputs.send_pr_feedback == 'true'
```

Comment:
...making this consistent with lines [121-124](https://github.com/bitwarden/browser-interactions-testing/pull/386/files#diff-afd5a779d08101c88e2ef7b12dc925599cb7830672243e2860b7f402914a3910R121-R124)